app/controllers/home_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  before_action :authenticate_user!
+  def dashboard
```

Comment:
add one line space between before_action and dashboard action @apeksha2697

Reply:
done